Patch to add Solaris 11.3 and Solaris 12 to the list of supported
OSes.  This patch has not yet been submitted upstream.

--- puppetlabs-apache-1.4.0/metadata.json.orig   2015-04-16 14:45:40.806485227 -0600
+++ puppetlabs-apache-1.4.0/metadata.json       2015-04-16 14:46:17.509385014 -0600
@@ -53,6 +53,13 @@
         "12.04",
         "14.04"
       ]
+    },
+    {
+      "operatingsystem": "Solaris",
+      "operatingsystemrelease": [
+        "11.3",
+        "12.0"
+      ]
     }
   ],
   "requirements": [
